src/runtime/tests/refiner-test.ts

```diff
@@ -427,113 +451,111 @@ describe('normalisation', () => {
-        let range1 = Range.infiniteRange();
-        // range1 = (-inf, +inf)
-        const range2 = new Range([Segment.closedClosed(0, 10), Segment.closedClosed(20, 30)]);
-        // range2 = [0, 10] U [20,30];
-        assert.isTrue(range2.isSubsetOf(range1));
-        range1 = new Range([Segment.closedClosed(0, 10), Segment.closedClosed(20, 30)]);
-        // range1 = [0, 10] U [20,30];
-        assert.isTrue(range2.isSubsetOf(range1));
-        range1 = new Range([Segment.closedClosed(0, 10), Segment.closedClosed(22, 30)]);
-        // range1 = [0, 10] U [22,30];
-        assert.isFalse(range2.isSubsetOf(range1));
-        range1 = new Range();
-        // range1 = [];
-        assert.isTrue(range1.isSubsetOf(range2));
-        range1 = new Range([Segment.closedOpen(0, 10), Segment.closedClosed(20, 30)]);
-        // range1 = [0, 10) U [20,30];
-        assert.isTrue(range1.isSubsetOf(range2));
-    });
-    it('tests the difference of ranges', () => {
-        let range1 = Range.infiniteRange();
-        // range1 = (-inf, +inf)
-        let range2 = new Range();
-        range2 = new Range([Segment.closedClosed(0, 10), Segment.closedClosed(20, 30)]);
-        // range2 = [0, 10] U [20,30];
-        let diff = Range.difference(range1, range2);
-        // diff = (-inf, 0) U (10,20) U (30, inf)
-        assert.isTrue(diff.equals(new Range([Segment.openOpen(Number.NEGATIVE_INFINITY, 0), Segment.openOpen(10, 20), Segment.openOpen(30, Number.POSITIVE_INFINITY)])));
-        range1 = new Range([Segment.closedOpen(0, 20), Segment.openClosed(40, 50)]);
-        // range1 = [0,20) U (40, 50]
-        range2 = new Range([Segment.openOpen(0, 5), Segment.closedOpen(7, 12), Segment.closedClosed(15, 43), Segment.openClosed(45, 50)]);
-        // range2 = (0,5) U [7,12) U [15, 43] U (45, 50]
-        diff = Range.difference(range1, range2);
-        // diff = [0, 0] U [5,7) U [12,15) U (43, 45]
-        assert.isTrue(diff.equals(new Range([Segment.closedClosed(0, 0), Segment.closedOpen(5, 7), Segment.closedOpen(12, 15), Segment.openClosed(43, 45)])));
-    });
-    it('tests the complement of ranges', () => {
-      let range = new Range([Segment.closedClosed(0, 10)]);
-      // range =  [0,10]
-      let complement = Range.complementOf(range);
-      // complement = (-inf, 0) U (10, inf)
-      assert.isTrue(complement.equals(new Range([Segment.openOpen(Number.NEGATIVE_INFINITY, 0), Segment.openOpen(10, Number.POSITIVE_INFINITY)])));
-      range = Range.booleanRange(0);
-      // range = [0,0]
-      complement = Range.complementOf(range);
-      // complement = [1,1]
-      assert.isTrue(complement.equals(Range.booleanRange(1)));
-    });
+  it('tests union operations on a range', () => {
+      const range1 = new Range();
+      // range1 = [];
+      range1.unionWithSeg(Segment.closedClosed(0, 10));
+      // range1 = [0, 10];
+      assert.isTrue(range1.equals(new Range([Segment.closedClosed(0, 10)])));
```

Comment:
Man, this file needs (needed?) quite a bit of updating!
For the rest of these issues, maybe we can file a tech-debt bug for later?

Reply:
I agree, though there is significantly less when viewed with 'Hide whitespace changes' on (this option can be found under the little cog above the file view and hides about 800 lines of changes put to re-indent it).